### ISSUE TITLE
Remove the always-online requirement

### ISSUE THREAD

**Issue by RichardFourie** [a85/POSTMan-Chrome-Extension#10254]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

The always-online requirement is probably the biggest mistake made in v8. When my internet goes down or when it is unstable, this app becomes useless, and no, scratchpad is not a solution to this.

### Describe the solution you'd like

Basically how it worked in v7. Have my work stored locally while I work (online or offline) and then synced to the cloud when I'm online.

**Comment by twocs** [comment]
This is such an important request. We should test an API in all network conditions, particularly corporate infrastructure, not only those where public routes to Postman are open.

**Comment by rosennej** [comment]
This day and age attention should be given to cyber security aspects. We do not want to expose internal APIs to the network.

**Comment by xpol** [comment]
Why do I have to connect to your * cloud when I want to view my postman requests and scripts?

I think it's time to create a similar product called **Localman**.

Thumb Up if you guys also want to switch to a locally only solution.

**Comment by skitchbeatz** [comment]
> Why do I have to connect to your * cloud when I want to view my postman requests and scripts?
> 
> I think it's time to create a similar product called **Localman**.
> 
> Thumb Up if you guys also want to switch to a locally only solution.

The postman cloud should be there as a backup & sync solution (IF the user decides to sync). It shouldn't be the only way to access previously created collections and environments. This limitation is bonkers

**Comment by vlad-deyneko** [comment]
@postman-support any progress on this?

I've already found a few topics related to application functionality without stable Internet connection, and I do not see a quick and easy solution to this. Personally, I sometimes need to work from my house, which is located close to the objects that have a significant impact on cellular network. Although, I can usually manage to somehow send requests to API, the workflow for testing a local API (on Docker) is just a mess. In addition, application is becoming useful after a minute or two after being started (because of all synchronization requests).

Is it possible for Postman to work offline or I should switch to another product?

**Comment by AndreBritoGit** [comment]
> @postman-support any progress on this?
> 
> I've already found a few topics related to application functionality without stable Internet connection, and I do not see a quick and easy solution to this. Personally, I sometimes need to work from my house, which is located close to the objects that have a significant impact on cellular network. Although, I can usually manage to somehow send requests to API, the workflow for testing a local API (on Docker) is just a mess. In addition, application is becoming useful after a minute or two after being started (because of all synchronization requests).
> 
> Is it possible for Postman to work offline or I should switch to another product?

you can use an older version
 https://go.pstmn.io/dl-win64-v7-latest

**Comment by rosennej** [comment]
As it seems the Postman is determined not to support an offline mode, it seems they are determined to drive all their security aware customers to alternative products. This a pity, but the is no other option.

**Comment by arlemi** [comment]
You can use the Scratchpad to use Postman without login-in/syncing: https://learning.postman.com/docs/getting-started/using-scratch-pad/

**Comment by AndreBritoGit** [comment]
why would i do that when i have sync, log in and if the connection is down I can still work on version 7?

**Comment by rosennej** [comment]
> You can use the Scratchpad to use Postman without login-in/syncing: https://learning.postman.com/docs/getting-started/using-scratch-pad/

This a rather lame solution.

**Comment by rosennej** [comment]
> why would i do that when i have sync, log in and if the connection is down I can still work on version 7?

This too is a lame solution.

**Comment by mmmasent** [comment]
This is a major setback in the application. This is a must-have. This seems akin to Microsoft and other game companies requiring a login to play single player games -- this is not logical. I should be able to run tests locally especially since my internet is a cellular one due to being in very rural area with no real access to cable line alternatives. i.e. Postman is now specifically telling individuals with access to subpar internet that they aren't a valued user.

I'll be honest, the need for an offline support mode is so great, this setback is making me consider creating my own personal solution if others do not exist with the needed functionality.

Offline mode should ALWAYS be considered in professional products, though that is my opinion. What happens if the Cloud goes down? Or is not desired? Woah, someone doesn't want to use the cloud? Say, what!? The cloud isn't an end-all, and it shouldn't be required when it is not necessary.

**Comment by lgg** [comment]
Let's be honest - they won't remove 'always online requirement'. It's their business-model, they push us to use their online services, where they collect and analyze data to sell it (read Terms of Usage carefully).

So I recommend everyone just to pick up an alternative:

- https://github.com/hoppscotch/hoppscotch
- https://github.com/rangav/thunder-client-support
- https://github.com/httpie/httpie
- https://github.com/SmartBear/soapui

Goodbye Postman, it was a good story. Like always - money rush ruined a good product.

**Comment by arlemi** [comment]
@lgg There is no "always online requirement" as I've mentioned in my previous comment: https://github.com/postmanlabs/postman-app-support/issues/10254#issuecomment-1015649165

**Comment by benjamin-lang** [comment]
@arlemi That is only partially true, I understand the business model with the move to the cloud and it's really great for private usage or small companies. But when it comes to big companies with strict regulatory requirements it's simply not allowed to use a tool, which syncs your internal api data in a cloud.

We for example, would really like to use postman within our company, but as long as there is no full offline mode for rest/grpc requests, it will never happen. (sadly)

**Comment by arlemi** [comment]
@benjamin-lang I'd recommend reaching out to sales and have a chat anyway, we already work with many big companies (cf [our case studies](https://www.postman.com/case-studies/) for a small subset) and might be able to get something to work for you too. 😄

**Comment by mmmasent** [comment]
Based on your insistence that scratchpad must be used, and is the only solution considered by postman, I've created a separate issue in regards to a seamless scratch pad and "cloud" workspace experience. Thank you for your time/comments.

**Comment by james2432** [comment]
> You can use the Scratchpad to use Postman without login-in/syncing: https://learning.postman.com/docs/getting-started/using-scratch-pad/

this aged as well as milk considering they are removing scratchpad in September

**Comment by Shijikori** [comment]
I refuse to update my software. I still have an installer for Postman 9 and that is what I will use. I don't want Yet-Another-Account to do my work. I mess with enough credentials already.  I don't want your cloud stuff for API requests, thank you.

**Comment by DSchau** [comment]
Hey folks!

We've heard the feedback here. With the deprecation of Scratchpad, we instead recommend users with these kinds of requirements -- e.g. not syncing to Postman cloud, no storing of local assets, etc. -- to use the Lightweight API Client, which  we announced in May 2023.

You can learn more about the [Lightweight API Client here](https://learning.postman.com/docs/getting-started/basics/using-api-client/).

I do understand that the spirit of the original request is around offline mode and then syncing back when online with the _main_ application, but we do not have plans to pursue that route at this time. In the future if we do, we will be sure to let you all know.

Until then, thank you for the feedback. It might not seem like it when we close a request, but we do sincerely appreciate it and use all of this feedback to learn and improve.

**Comment by ijash-work** [comment]
> I refuse to update my software. I still have an installer for Postman 9 and that is what I will use. I don't want Yet-Another-Account to do my work. I mess with enough credentials already. I don't want your cloud stuff for API requests, thank you.

just move to insomnia or hoppscotch. their decision on using "always online" mode were a total FAIL. and the wont listen to any "offline" idea. just look at https://github.com/postmanlabs/postman-app-support/issues/10254#issuecomment-1879377173 reply

**Comment by erikdw** [comment]
🖕

**Comment by dmitrij9992905** [comment]
As for embedded developer who needed the Postman for validation APIs on devices such as ESP32-based device to setup WiFi and other settings thru it's own WiFi AP, where there is no Internet at all. Today, I see that Postman refused to send request to device thru its AP until I gave the Postman Internet again. CURL helped me in that bit I have collections.
Well, I need to search and migrate to another software like yours.
Farewell, have a nice stay
